_data/simple-icons.json

```
@@ -1415,6 +1420,11 @@
             "hex": "00D6D6",
             "source": "https://fandomdesignsystem.com/"
         },
+        {
+            "title": "Fastify",
+            "hex": "000000",
+            "source": "https://www.fastify.io/"
```

Since the website itself does not have a clear SVG, and this repo is probably where icons will be updated in the future anyway I suggest we update the source 👍 

```suggestion
            "source": "https://github.com/fastify/graphics"
```

---

Done